packit/utils.py

```diff
@@ -250,3 +254,39 @@ def get_namespace(url: str) -> str:
         return namespace
     except IndexError:
         return None
+
+
+def get_real_sync_files(list_to_sync: list) -> List:
+    logger.debug("Packit synced files %s", list_to_sync)
+    real_files = []
+    for sync in list_to_sync:
+        src = glob.glob(sync["src"])
+        real_files.append({"src": src, "dest": sync["dest"]})
+    logger.debug(f"Real synced file {real_files}")
+    return real_files
```

Comment:
`get_real_sync_files()` is a bit ambiguous name. I'm thinking about some better name:
* `get matched_sync_files()`
*  `get_globbed_sync_files()`
* `get_wildcard_resolved_sync_files()`
* ?

Reply:
Well, `get_wildcard_resolved_sync_files` sounds good to me.